app/server/templates/admin/label.html

```diff
@@ -36,42 +35,61 @@
             <a class="tag is-medium" v-bind:style="{ color: textColor, backgroundColor: backgroundColor }">
               [[ labelText ]]
             </a>
-            <span class="tag is-medium">[[ selectedShortkey ]]</span>
+            <span class="tag is-medium"><code>[[ previewKeys ]]</code></span>
```

Comment:
I think this line is somewhat redundant.
In label.js, `previewKeys` is almost equivalent to `simpleShortcut` filter.
So I think it is better to use `[[ combineKeys | simpleShortcut ]]` instead of using `previewKeys`.

Reply:
Thanks, this can be simplified!